### PR TITLE
fix(filemanager): Binaries can be downloaded from a custom CDN

### LINKED ISSUE #96: --alternate_cdn option no longer works
Looks like `webdriver-manager` no longer honours the `--alternate_cdn` option.

Executing:
```
λ ~/ webdriver-manager update --alternate_cdn=http://johanneshoppe.github.io/webdriver-manager-alternate-cdn/
```

Results in `webdriver-manager` going to the regular CDN, ignoring the provided option:

```
[20:56:08] I/downloader - selenium standalone: downloading version 2.53.1
[20:56:08] I/downloader - curl -o /path/selenium-server-standalone-2.53.1.jar https://selenium-release.storage.googleapis.com/2.53/selenium-server-standalone-2.53.1.jar
```

### PR DESCRIPTION
**Pull request by jan-molak**
closes #96

### CODE REVIEW

**Comment by jan-molak** [comment]
Just signed the CLA

File: lib/files/file_manager.ts [hunk 1]
```diff
@@ -54,25 +54,25 @@ export class FileManager {
    * @param osType The operating system.
```
Comment (cnishina) [inline lib/files/file_manager.ts]: update the parameters for the JSDoc

File: lib/files/file_manager.ts [hunk 2]
```diff
@@ -82,7 +82,9 @@ export class FileManager {
    * for the system.
    * @returns A binary map that is available for the operating system.
```
Comment (cnishina) [inline lib/files/file_manager.ts]: update JSDoc `@param` for `alternativeCDN`

**Comment by jan-molak** [comment]
Thanks for reviewing @cnishina, please see the updated PR addressing the points you've raised.

**Comment by cnishina** [comment]
LGTM.

**Pull request merged**